go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cosmos/gaia/v4
 
-go 1.15
+go 1.16
```

Comment:
You don't need to bump this

Reply:
I meant that the `go` tool automatically bumps this dependency when compiling if necessary

Reply:
Thanks for your insight into go's implicit behaviour! 🙏 

I'll stick with 1.16 so that it's consistent and explicit.